lib/concurrent/jruby_atom.rb

```diff
@@ -0,0 +1,223 @@
+if Concurrent.on_jruby?
+
+  require 'concurrent/collection/copy_on_notify_observer_set'
+  require 'concurrent/concern/observable'
+  require 'java'
+
+  java_import 'java.util.concurrent.atomic.AtomicReference'
```

Comment:
Out of curiosity, how big is the perf hit of using Concurrent::Reference versus the Java one?